netCDF4/_netCDF4.pyx

```diff
@@ -1186,8 +1257,17 @@ cdef _get_types(group):
                     #print 'WARNING: unsupported VLEN type, skipping...'
                     continue
                 vltypes[name] = vltype
-                pass
-    return cmptypes, vltypes
+            elif classp == NC_ENUM: # an enum type
+                name = namstring.decode(default_encoding,unicode_error)
+                # read the Enum type info from the file,
+                # create a EnumType instance from it.
+                try:
+                    enumtype = _read_enum(group, xtype)
+                except KeyError:
+                    #print 'WARNING: unsupported Enum type, skipping...'
```

Comment:
When does this happen? Generally it's better to raise exception than pass them silently....